include/librealsense2/h/rs_option.h

```diff
@@ -148,6 +148,7 @@ extern "C" {
     /** \brief For setting the camera_mode option */
     typedef enum rs2_sensor_mode
     {
+        RS2_SENSOR_MODE_QVGA,
```

Comment:
This change breaks the existing API.
Should be appended before COUNT

Reply:
Fixed